frontend/src/scenes/actions/ActionPlugins.tsx

```diff
@@ -1,67 +1,101 @@
-import { LemonButton, LemonTable } from '@posthog/lemon-ui'
+import { IconPlus } from '@posthog/icons'
+import { LemonButton, LemonSkeleton, LemonTable } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
+import { useFeatureFlag } from 'lib/hooks/useFeatureFlag'
 import { LemonTableLink } from 'lib/lemon-ui/LemonTable/LemonTableLink'
 import { useEffect } from 'react'
 import { actionLogic } from 'scenes/actions/actionLogic'
 import { PluginImage } from 'scenes/plugins/plugin/PluginImage'
 import { urls } from 'scenes/urls'
 
-import { PipelineNodeTab, PipelineStage } from '~/types'
+import { EntityFilter, PipelineNodeTab, PipelineStage } from '~/types'
 
 export function ActionPlugins(): JSX.Element | null {
-    const { matchingPluginConfigs } = useValues(actionLogic)
+    const { action, matchingPluginConfigs, matchingPluginConfigsLoading } = useValues(actionLogic)
     const { loadMatchingPluginConfigs } = useActions(actionLogic)
 
+    const pluginFilters = !!useFeatureFlag('PLUGINS_FILTERING')
```

Comment:
What is this flag and why isn't it a constant in `constants.tsx`?

Reply:
Yeah easier to work with than a constant. Tbh I might remove all this stuff as it was oriented around plugins filtering which we will likely just ditch as a direction. Then I can add it in later properly for the new stuff

Reply:
Removed the whole file for now. Will come back to that idea later